extensions/simpread/src/sr.tsx

```diff
@@ -0,0 +1,190 @@
+import {
+  ActionPanel,
+  OpenInBrowserAction,
+  Icon,
+  ImageMask,
+  List,
+  PushAction,
+  Detail,
+  getLocalStorageItem,
+  getPreferenceValues,
+  showToast,
+  ToastStyle,
+} from '@raycast/api';
+import { readFileSync } from 'fs';
+import { useEffect, useState } from 'react';
+import { URL } from 'url';
+
+const TurndownService = require('turndown');
+
+interface Preferences {
+  path: string;
+}
+
+interface State {
+  items?: [];
+  error?: Error;
+}
+
+interface Unread {
+  id: number;
+  idx: number;
+  url: string;
+  title: string;
+  create: string;
+  icon: string;
+  favicon?: string;
+  markdown?: string;
+  desc?: string;
+  note?: string;
+  tags?: string[];
+  annotations: Annote[];
+}
+
+interface Annote {
+  id: number;
+  type: string;
+  html: string;
+  text: string;
+  note?: string;
+  tags?: string[];
+}
+
+function getDetail(unread: Unread) {
+  const turndown = new TurndownService(),
+    fmtDate = (date: Date) => {
+      const format = (value: any) => (value = value < 10 ? '0' + value : value);
+      return (
+        date.getFullYear() +
+        '年' +
+        format(date.getMonth() + 1) +
+        '月' +
+        format(date.getDate()) +
+        '日 ' +
```

Comment:
At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension